.github/workflows/create_pr_workflow.yml

```diff
@@ -0,0 +1,48 @@
+name: Create Pull Request
+
+on: 
+    workflow_dispatch:
+    release:
+      types: [edited]
+
+jobs:
+
+  build:
+    env:
+      REPOSITORY_NAME: "node-versions"
+      BRANCH_NAME: "update-versions-manifest-file"
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: env
+      run: Get-ChildItem -Path Env:\
+      shell: pwsh
+
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Create versions-manifest.json
+      run: |
+        ./helpers/packages-generation/manifest-generator.ps1 -GitHubRepositoryOwner "${{github.repository_owner}}" `
+                                                             -GitHubRepositoryName "$env:REPOSITORY_NAME"`
+                                                             -GitHubAccessToken "${{secrets.GITHUB_TOKEN}}"`
+                                                             -OutputFile "./versions-manifest.json"`
+                                                             -ConfigurationFile "./config/node-manifest-config.json"
+      shell: pwsh
```

Comment:
Let's move `shell` prop to between `name` and `run`

Reply:
Moved it